autopush/web/webpush.py

```diff
@@ -204,6 +204,48 @@ def reject_encryption_key(self, data, original_data):
             )
 
 
+class WebPushCrypto06HeaderSchema(Schema):
+    """Validate webpush Message Encryption
+
+    Uses draft-ietf-webpush-encryption-06 rules for validation
+
+    """
+    content_encoding = fields.String(
+        required=True,
+        load_from="content-encoding",
+        validate=OneOf(["aes128gcm"])
+    )
+    crypto_key = fields.String(
+        required=True,
+        load_from="crypto-key",
+    )
+
+    @validates("crypto_key")
+    def validate_crypto_key(self, value):
+        """Must contain a dh value
+
+        This draft ignores the salt and keyid that may be in
+        the header for values that are specified in the content
+        text.
+
+        """
+        dh = CryptoKey.parse_and_get_label(value, "dh")
+        if not dh or not VALID_BASE64_URL.match("dh"):
+            raise InvalidRequest("Invalid dh value in Encryption-Key header",
+                                 status_code=400,
+                                 errno=110)
+
+    @validates_schema(pass_original=True)
```

Comment:
that's copied from [WebPushCrypto04HeaderSchema](https://github.com/mozilla-services/autopush/pull/775/files/9784937e00d51921cead691a412dc63a77ee6590#diff-ccbbd52ff880cf4ddb71b18d126d189cR196) Should both be modified?